tests/testthat/test-param-labels.R

```diff
@@ -83,55 +83,52 @@ for (MODEL_PICK in MODEL_PICKS) {
 
 
 # test param_labels against tidynm reference tibbles (testing bbi_nonmem_model dispatch)
-if (Sys.getenv("METWORX_VERSION") == "" && Sys.getenv("DRONE") != "true") {
-  skip("param_labels only runs on Metworx or Drone")
-} else {
-  withr::with_options(list(rbabylon.bbi_exe_path = read_bbi_path()), {
-
-    for (MODEL_PICK in MODEL_PICKS) {
-      .mod_id <- MODEL_PICK$mod_id
-
-      test_that(glue("param_labels.bbi_nonmem_model() %>% apply_indices() matches tidynm reference for {.mod_id}"), {
-        # get reference df from tidynm test data
-        ref_df <- readRDS(file.path(REF_DIR, "param-labels", glue("{.mod_id}_PARAMTBL.rds")))[[1]]
-        names(ref_df) <- names(ref_df) %>% tolower()
-
-        # get param df with rbabylon::model_summary()
-        if (fs::file_exists(file.path(PL_MODEL_DIR, glue("{.mod_id}.yaml")))) fs::file_delete(file.path(PL_MODEL_DIR, glue("{.mod_id}.yaml")))
-
-        .mod <- new_model(file.path(PL_MODEL_DIR, .mod_id))
-
-        on.exit({ if (fs::file_exists(file.path(PL_MODEL_DIR, glue("{.mod_id}.yaml")))) fs::file_delete(file.path(PL_MODEL_DIR, glue("{.mod_id}.yaml"))) })
-
-        .param_df <- .mod %>% rbabylon::model_summary() %>% param_estimates()
-        names(.param_df) <- names(.param_df) %>% tolower()
-
-        # make label df
-        .label_df <- .mod %>% param_labels()
-
-        # join to constuct full parameter table
-        suppressSpecificWarning({
-          .new_df <- inner_join(.param_df,
-                                .label_df %>% apply_indices(.omega = MODEL_PICK$omega, .sigma = MODEL_PICK$sigma) %>% select(-param_type),
-                                by = SUMMARY_PARAM_NAMES)
-        }, .regexpr = "Column .+ has different attributes on LHS and RHS of join")
-
-        # join against reference to make sure they're the same
-        ref_df <- ref_df %>% mutate(!!SUMMARY_PARAM_NAMES := ifelse(param == "THETA",
-                                                                    paste0(param, var1),
-                                                                    paste0(param, "(", var1, ",", var2, ")")))
-
-        join_df <- ref_df %>%
-          full_join(.new_df, by = names(.label_df))
-
-        # tests
-        expect_equal(nrow(join_df), nrow(ref_df))
-        expect_equal(nrow(join_df), nrow(.new_df))
-        expect_equal(join_df$value, join_df$estimate, tolerance = 0.01)
-        expect_equal(join_df$se, join_df$stderr, tolerance = 0.01)
-        expect_equal(join_df$c, join_df$random_effect_sd, tolerance = 0.01)
-        expect_equal(join_df$cse, join_df$random_effect_sdse, tolerance = 0.01)
-      })
-    }
-  })
-}
+withr::with_options(list(rbabylon.bbi_exe_path = read_bbi_path()), {
```

Comment:
all of these "changes" below are just un-indenting because I took this out of an `else` block.